crates/fuel-core/src/schema/dap.rs

```diff
@@ -456,7 +456,7 @@ impl DapMutation {
         match checked_tx {
             CheckedTransaction::Script(script) => {
                 let ready_tx = script
-                    .into_ready(GAS_PRICE, gas_costs, fee_params)
+                    .into_ready(GAS_PRICE, gas_costs, fee_params, None)
```

Comment:
I didn't add a todo because in this case it seems None is the right value, but please check @AurelienFT

Reply:
I think we can use `vm_database.block_height()?` I will do it.